.github/workflows/ci-frontend.yml

```diff
@@ -0,0 +1,30 @@
+name: Perform Lint Checks on Frontend
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the development branch
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "frontend_ci"
+  frontend_ci:
```

Comment:
could you rename this frontend_lint just to be uniform with the other ticket

Reply:
Done, thanks!